README.md

```diff
@@ -1 +1,281 @@
-# neutrino-preset-mozilla-rpweb
\ No newline at end of file
+# Neutrino Mozilla R&P Web Preset
+[![NPM version][npm-image]][npm-url] [![NPM downloads][npm-downloads]][npm-url] [![Join Slack][slack-image]][slack-url] 
+
+`neutrino-preset-mozilla-rpweb` is a Neutrino preset that supports creating React projects and linting them with
+Airbnb's base ESLint config, following the [Airbnb styleguide](https://github.com/airbnb/javascript). This preset is
+used for web projects within Mozilla's Release and Productivity team.
+
+## Features
+
+From `neutrino-preset-react`:
+- Zero upfront configuration necessary to start developing and building a React web app
+- Modern Babel compilation adding JSX and object rest spread syntax.
+- Support for React Hot Loader
+- Write JSX in .js or .jsx files
+- Extends from [neutrino-preset-web](https://neutrino.js.org/customization/presets/neutrino-preset-web)
+  - Modern Babel compilation supporting ES modules, last 2 major browser versions, and async functions
+  - Webpack loaders for importing HTML, CSS, images, icons, and fonts
+  - Webpack Dev Server during development
+  - Automatic creation of HTML pages, no templating necessary
+  - Hot module replacement support
```

Comment:
1. Should we make mention of Tree-shaking? It's a pretty huge feature :)
2. Capitalize **H**ot **M**odule **R**eplacement